jsk_tools/src/test_topic_published.py

```diff
@@ -3,19 +3,41 @@
 
 import re
 import sys
+import time
 import unittest
 
 from nose.tools import assert_true
-from nose.tools import assert_false
 
 import rospy
-from jsk_tools.sanity_lib import TopicPublishedChecker
+import rostopic
 
 
 PKG = 'jsk_tools'
 NAME = 'test_topic_published'
 
 
+# https://github.com/ros/ros_comm/blob/kinetic-devel/tools/rostest/nodes/publishtest # NOQA
+class PublishChecker(object):
+    def __init__(self, topic_name, timeout, negative):
+        self.topic_name = topic_name
+        self.negative = negative
+        self.deadline = rospy.Time.now() + rospy.Duration(timeout)
+        msg_class, _, _ = rostopic.get_topic_class(
+            rospy.resolve_name(topic_name), blocking=True)
+        self.msg = None
+        self.sub = rospy.Subscriber(topic_name, msg_class, self._callback)
+
+    def _callback(self, msg):
+        self.msg = msg
+
+    def assert_published(self):
```

Comment:
if you name `assert_published`, it is better and easier to understand to return `True` when msg is published and `False` when it isn't.